script.js

```diff
@@ -10,3 +10,215 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 5,
+    title: 'Multi-Post Stories Gain+Glory Gain+Glory',
+    snapshot: 'Images/assets/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    supdescription: 'Keeping track of hundreds of components',
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 6,
+    title: 'Multi-Post Stories Gain+Glory',
+    snapshot: 'Images/assets/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    supdescription: 'Keeping track of hundreds of components',
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+];
+
+projects.forEach((project) => {
+  const bodySection = document.querySelector('.project-cards');
+  const section = document.createElement('section');
+  section.className = 'projects';
+  section.innerHTML = `<div class=""project-cards">
+  <img class="project-card" src=${project.snapshot} alt="Snap image for background">
+  </div>
+<div class="project-card_details">
+  <h2 class="title">${project.title}</h2>
+  
+  <div class="stacks">
+  <button class="stacks1" type="button" name="button">${project.skills[0]}</button>
+  <button class="stacks1" type="button" name="button">${project.skills[1]}</button>
+  <button class="stacks1" type="button" name="button">${project.skills[2]}</button>
+  <button class="stacks1" type="button" name="button">${project.skills[3]}</button>
+  </div>
+  
+  <div class="action-btn-container">
+  <button class="green-button" type="button" id="btn-${project.id}" name="button">See project</button>
+  
+  </div>
+</div>
+`;
+
+  bodySection.append(section);
+});
+const body = document.querySelector('section');
+projects.forEach((project) => {
+  const popupWindow = document.createElement('div');
+  popupWindow.className = 'popup-window';
+  popupWindow.id = `card-${project.id}`;
+  popupWindow.innerHTML = `
+  <section class="project-card">
+  <div class="pop-header">
```

Comment:
- [ ] Kindly make sure you stick to the original template as much as possible. Your typos (fonts size, weight), color, spacing are supposed to be similar with the original design.
As you may notice the difference between your popup modal and the original design 
### Original design :point_down: 
![image](https://github.com/newtonah0/Mobile-Portfolio/assets/97100091/dad2f0b9-2579-4978-bb47-6f871b2ea789)
### Your design :point_down: 
![image](https://github.com/newtonah0/Mobile-Portfolio/assets/97100091/21a29b1b-42b9-4498-a998-ffb4088689eb)

Reply:
![image](https://github.com/newtonah0/Mobile-Portfolio/assets/80107996/57e0c709-df63-4b68-9886-d7c19ed0e78c)
my design and my images is work on my browser?

Reply:
![image](https://github.com/newtonah0/Mobile-Portfolio/assets/80107996/9b8faa94-ebd3-4d9f-ab88-57449602b930)
and that in the desktop version.

Reply:
![image](https://github.com/newtonah0/Mobile-Portfolio/assets/80107996/3b915a70-205b-4a69-8be6-97ea31543eaa)
it's show in my design? @christianbiring1